runtime/lua/vim/_extui/shared.lua

```diff
@@ -42,12 +42,13 @@ function M.check_targets()
       or not api.nvim_win_is_valid(M.wins[type])
       or not api.nvim_win_get_config(M.wins[type]).zindex -- no longer floating
     then
+      local winborder = vim.o.winborder
```

Comment:
```suggestion
      local winborder = vim.o.winborder == '' and 'single' or vim.o.winborder
```

Reply:
I can do it, but stylua breaks it up into two lines which I didn't like